keep/rulesengine/rulesengine.py

```diff
@@ -0,0 +1,127 @@
+import hashlib
+import logging
+
+import celpy
+
+from keep.api.core.db import create_alert as create_alert_db
+from keep.api.core.db import get_rules as get_rules_db
+from keep.api.core.db import run_rule as run_rule_db
+from keep.api.models.alert import AlertDto
+
+
+class RulesEngine:
+    def __init__(self, tenant_id=None):
+        self.tenant_id = tenant_id
+        self.logger = logging.getLogger(__name__)
+
+    def run_rules(self, events: list[AlertDto]):
```

Comment:
Logs are a bit strange here.
You are "Running rules" twice:
1. once when you validate if an event is relevant to some rule
2. when you actually run the rule.

perhaps logs should be "validating rule relevancy" and "running rule"?